views/layouts/default.html.erb

```diff
@@ -29,13 +29,14 @@
             <nav class="pd-b">
               <a href="/" title="Welcome">Welcome</a>
               <a href="/#sec-hardfork" title="Bigger&nbsp;blocks">Bigger&nbsp;blocks</a>
+              <a href="/onboard.html" title="Who's Onboard">Who's&nbsp;Onboard</a>
```

Comment:
@NxtChg Yup, something like that. This issue isn't much for desktop screens, but rather for mobiles and tablets.